images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -270,3 +270,14 @@ function Get-PacmanVersion {
     $pacmanVersion = $matches[0]
     return "- Pacman $pacmanVersion"
 }
+
+function Get-YAMLLintVersion {
+    $yamlVersion = (yamllint --version).Split()[1]
+    $yamlPath = (Get-Command -Name yamllint).Path
```

Comment:
Why we want to print tool path to the documentation if tool is in PATH?

Reply:
Fixed.